hazelcast/src/main/resources/hazelcast-full-example.xml

```diff
@@ -89,7 +92,9 @@
         You can also set your license key programmatically; please refer to
         http://docs.hazelcast.org/docs/latest/manual/html-single/index.html#setting-the-license-key
     -->
-    <license-key>Your Hazelcast Enterprise or Enterprise HD License Key</license-key>
+    <license-key>Your Hazelcast Enterprise or Enterprise HD License
```

Comment:
Can you avoid reformatting in this file and the YAML example?

Reply:
fixed both.